### PR TITLE
firewall Ipv6 test

### PR DESCRIPTION
**Pull request by outscale-mgo**

### CODE REVIEW

File: tests/firewall/tests.c [hunk 1]
```diff
@@ -133,25 +175,28 @@ static void firewall_filter_rules(enum pg_side dir)
 	uint64_t bit;
 	uint16_t packet_count;
 	struct ip *ip;
+	struct ip6_hdr *ipv6;
 	struct ether_hdr *eth;
+	FILE *fw_pcap = fopen("fw.pcap", "w+");
```
Comment (outscale-jju) [inline tests/firewall/tests.c]: should be removed :)

File: tests/firewall/tests.c [hunk 2]
```diff
@@ -133,25 +175,28 @@ static void firewall_filter_rules(enum pg_side dir)
...
-	gen = pg_packetsgen_new("gen", 2, 2, pg_flip_side(dir), packets, nb, &error);
+	gen = pg_packetsgen_new("gen", 2, 2, pg_flip_side(dir),
+				packets, nb, &error);
 	g_assert(!error);
 	fw = pg_firewall_new("fw", 2, 2, PG_NONE, &error);
 	g_assert(!error);
 	col = pg_collect_new("col", 2, 2, &error);
 	g_assert(!error);
+	print = pg_print_new("print", 1, 1, fw_pcap,
+			     PG_PRINT_FLAG_PCAP, 0, &error);
```
Comment (outscale-jju) [inline tests/firewall/tests.c]: print brick is usefull for debug but less for tests :)

**Comment by outscale-mgo** [comment]
ping :)
Even if firewall still miss some more complex test, we can already integrate this one.

**Comment by outscale-jju** [comment]
small fixes and merged

**Pull request closed without merging**